string name
string category
string size
string[] extras
float32 price
